added in forking to a new configuration (#991)
timescale/docs: cloud/operations.md

complete the transition before you start forking.
</highlight>

<highlight type="warning">
Forks only have data up to the point when the original service was forked. Any

> data written to the original service after the time of forking will not appear 

Reviewer: ```suggestion
data written to the original service after the time of forking does not appear 
```

> in the fork. If you intend for the fork to assume operations from the original

Reviewer: ```suggestion
in the fork. If you want the fork to assume operations from the original
```

> service, you may want to pause your main service before forking to avoid any

Reviewer: ```suggestion
service, you can pause your main service before you fork and avoid any
```

> service, you may want to pause your main service before forking to avoid any
> data discrepancy between services.

Reviewer 2: ```suggestion
service, pause your main service before forking to avoid any
data discrepancy between services.
```

1.  Navigate to the `Operations` tab.
1.  In the `General` section, click `Fork service`. In the dialog, confirm by
    clicking `Fork service`. The forked service takes a few minutes to start.

> 1.  If you would like to change the configuration of your fork, click the
>     `Advanced options` link in the dialog.

Reviewer 2: ```suggestion
1.  To change the configuration of your fork, click
    `Advanced options`. You can set different compute and
    storage options, separate from your original database.
```
